servicebroker_node/test/fat/index_test.js

```diff
@@ -0,0 +1,34 @@
+var supertest = require("supertest");
+var should = require("should");
+var fs = require('fs');
+var path = require('path');
+
+// This agent refers to PORT where program is runninng.
+var settings = JSON.parse(fs.readFileSync( path.join (__dirname, '../../config/settings.json'), 'utf8'));
+var port = process.env.PORT || settings.port;
+var server = supertest.agent("http://localhost:"+port);
```

Comment:
We should be starting and stopping the server as needed for our testing.
See https://glebbahmutov.com/blog/how-to-correctly-unit-test-express-server/ as an example.